app/demo-geant-integration/SensitiveDetector.cc

```diff
@@ -0,0 +1,48 @@
+//----------------------------------*-C++-*----------------------------------//
+// Copyright 2022 UT-Battelle, LLC, and other Celeritas developers.
+// See the top-level COPYRIGHT file for details.
+// SPDX-License-Identifier: (Apache-2.0 OR MIT)
+//---------------------------------------------------------------------------//
+//! \file demo-geant-integration/SensitiveDetector.cc
```

Comment:
For demonstration purposes I think one is sufficient, but we can also move over the tracker/calo from your `acc01` example. Maybe if we move those over, we can add another option to mark which volumes are which types of detectors (since it would be problem-dependent)?

Reply:
Another option may be to template the current class with the name of sensitive detector and hit collection type associated with it.